### PR TITLE
Defiler lore change

### PR DESCRIPTION
**Pull request by Sebbe9123**
## About The Pull Request
Changes the defiler lore blurb from being something about impregnating marines to being something about it's role as a smoke screen maker and chem injector.

## Why It's Good For The Game
Makes people not confused when you hit anchient.

## Changelog
:cl:
fix: Changes anchient lore blurb about defiler to fit more with gameplay.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/carbon/xenomorph/castes/defiler/castedatum_defiler.dm [hunk 1]
```diff
@@ -122,7 +122,7 @@
 /datum/xeno_caste/defiler/ancient
 	upgrade_name = "Ancient"
 	caste_desc = "Being within mere eyeshot of this hulking, dripping monstrosity fills you with a deep, unshakeable sense of unease."
-	ancient_message = "We are the ultimate alien impregnator. We will infect the marines, see them burst open before us, and hear the gleeful screes of our larvae."
+	ancient_message = "We're a bristling, living alien chemlab. Our body leaks noxious fumes that will slip through even the most powerful protections"
```
Comment (Pariah919) [inline code/modules/mob/living/carbon/xenomorph/castes/defiler/castedatum_defiler.dm:125]:
```suggestion
	ancient_message = "We're a bristling, living alien chemlab. Our body leaks noxious fumes that will slip through even the most powerful protections."
```

Comment (Sebbe9123) [reply code/modules/mob/living/carbon/xenomorph/castes/defiler/castedatum_defiler.dm:125]: Fuck, you right.

**Comment by InterroLouis** [comment]
Ancient not Anchient.

**Comment by Ragecrabbo** [comment]
Also you should write its not it's. Literally unmergable.

**Comment by Sebbe9123** [comment]
:agony:

**Review by Pariah919 — APPROVED** [review]
cool

**Comment by EvilMelonTheSecond** [comment]
Finally a reason to play defiler again

**Review by TiviPlus — APPROVED** [review]

**Pull request merged**